nova/core/roles/accounts/tasks/unix_create.yml

```diff
@@ -13,21 +13,34 @@
     name: "{{ item.username }}"
     password: "{{ item.password | default(random_generated_password if not save_secrets_to_vault else pregenerated_password) | password_hash('sha512') }}"
     shell: "{{ item.shell | default(unix_distro_shell_map[ansible_distribution] | default('/bin/bash')) }}"
-    groups: "{{ item.groups | default(root_group if item.username in admins_list else 'users') }}"
+    groups: "{{ item.groups | default(root_group if item.username in admin_accounts | map(attribute='username') else 'users') }}"
     skeleton: "{{ unix_distro_skel_map[ansible_distribution] | default('/etc/skel') }}"
     createhome: true
     append: true
-  loop: "{{ accounts }}"
+  loop: "{{ local_accounts_list }}"
   loop_control:
     label: "{{ item.username }}"
 
+- name: Adding account creation source flag...
```

Comment:
What do you think that instead of creating a file at user's home folder we include a note directly in the GECOS field (user-info) from /etc/passwd ?

Maybe in windows to find something similar is a bit challenge

Reply:
After reviewing the code, I think its easier and more similar to have same in all OS as you did it already